packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/common_blocs/push_notifications_bloc.dart

```diff
@@ -0,0 +1,38 @@
+{{> licence.dart }}
+
+import 'package:rx_bloc/rx_bloc.dart';
+import 'package:rxdart/rxdart.dart';
+
+import '../../lib_router/blocs/router_bloc.dart';
+import '../../lib_router/router.dart';
+import '../models/notification_model.dart';
+part 'push_notifications_bloc.rxb.g.dart';
+
+/// A contract class containing all events of the PushNotificationsBloC.
+abstract class PushNotificationsBlocEvents {
+  void tapOnEvent(NotificationModel event);
```

Comment:
Why did you remove this documentation comment?
```
  /// Event for handling opening push notifications from background
  /// or from in-app
```

Reply:
Removed by mistake 👍